tests/console/postgresql94server.pm

```diff
@@ -29,7 +29,13 @@ sub run() {
 
     # check the status
     assert_script_run "systemctl show -p ActiveState postgresql.service | grep ActiveState=active";
-    assert_script_run "systemctl show -p SubState postgresql.service | grep SubState=running";
+
+    if (get_var('VERSION', '12')) {    # loaded via init.d on SLES 12 GA
```

Comment:
what you meant here is check_var. get_var will always return something - *especially* if you give it a default.